src/workers/postUpdated.ts

```diff
@@ -166,13 +168,19 @@ const allowedFieldsMapping = {
     'description',
     'metadataChangedAt',
     'readTime',
-    'siteTwitter',
     'summary',
     'tagsStr',
     'toc',
   ],
 };
 
+const contentTypeFromPostType: Record<PostType, typeof Post> = {
+  [PostType.Article]: ArticlePost,
+  [PostType.Freeform]: FreeformPost,
+  [PostType.Share]: SharePost,
+  [PostType.Welcome]: WelcomePost,
+};
+
```

Comment:
@rebelchris we might wanna extract it to some common file immediately for reuse?

Reply:
Probably a good one for when we need it, not sure where else we evaluate based on types. 
It's something Yggdrasil uses, and not particularly the API at this point.

Probably another thing Protobuf would fix

Reply:
Yeah, or any kind of cross service standardisation 👍